packages/unlock-js/src/locksmithService.ts

```diff
@@ -71,10 +71,11 @@ export class LocksmithService {
    */
   async login(message: string, signature: string) {
     const headers = new Headers()
-    headers.set('content-type', 'application/json')
     const response = await this.request('/v2/auth/login', {
       method: 'POST',
-      headers,
+      headers: {
+        'Content-Type': 'application/json',
+      },
```

Comment:
I think you can keep the old code here. Just import Headers from `import fetch, { Headers } from 'cross-fetch'`. It is a polyfill issue.